cmd/cost/list.go

```diff
@@ -142,6 +182,87 @@ func listCostsUnderOU(OU *organizations.OrganizationalUnit, awsClient awsprovide
 	return nil
 }
 
+type AccountCost struct {
+	AccountID string
+	Cost      float64
+	Unit      string
+}
+
+type OUCost struct {
+	Costs   []AccountCost
+	OU      *organizations.OrganizationalUnit
+	options *listOptions
+}
+
+func (o *OUCost) getCost(awsClient awsprovider.Client) error {
+
+	accounts, err := getAccountsRecursive(o.OU, awsClient)
+	if err != nil {
+		return err
+	}
+
+	ops := &getOptions{
+		time:  o.options.time,
+		start: o.options.start,
+		end:   o.options.end,
+		ou:    o.options.ou,
+	}
+
+	for _, account := range accounts {
+		accCost := AccountCost{
+			AccountID: *account,
+			Unit:      "",
+			Cost:      0,
+		}
+		err = ops.getAccountCost(account, &accCost.Unit, awsClient, &accCost.Cost)
+		if err != nil {
+			return err
+		}
+		o.Costs = append(o.Costs, accCost)
+	}
+
+	sort.Slice(o.Costs, func(i, j int) bool {
+		return o.Costs[i].Cost < o.Costs[j].Cost
+	})
+
+	return nil
+}
+
+func (o OUCost) getSum() (sum float64, unit string, err error) {
+	sum = 0
+	for _, cost := range o.Costs {
+		sum += cost.Cost
+		if unit == "" {
+			unit = cost.Unit
```

Comment:
I guess this is a phase 1 thing, and if we get more it'll be implemented better

Reply:
more what? I think the current assumption is there is only USD. This is already an improvement since it will fail if not all numbers have the same currency.

Reply:
And if in some wierd case it's not, a future implementation will sum by currency

But as I said it's phase 2 of this (if we get the error alot)